### DISCUSSION TITLE
height maps

### DISCUSSION THREAD

**Discussion by zoslcne** [sh4rkman/SquadCalc#230] [Q&A] [answered]
How are these height maps made? Can they be accurate to buildings?

**Comment by sh4rkman** [comment, marked as answer]
They are from the game itself, but only hold terrain : no building/bridges : 
- https://github.com/sh4rkman/SquadCalc/wiki/Deducing-Altitude
- https://github.com/sh4rkman/SquadCalc/wiki/Adding-new-map
